modules/rest-api/pages/rest-set-up-services-existing-nodes.adoc

```diff
@@ -0,0 +1,151 @@
+= Assigning Services on an Existing Node
+
+:description: pass:q[Use the REST API to assign or remove services on an existing node in a cluster, and rebalance the cluster.]
+:page-topic-type: reference
+
+[abstract]
+{description}
+
+[#http-method-and-uri]
+
+=== HTTP Method and URI
+
+----
+POST /controller/rebalance
+----
+
+== Description
+
+Use the following REST API to add or remove non-Data services on the existing nodes in a cluster.
+
+[NOTE]
+====
+* Follow the xref:manage:manage-nodes/modify-services-and-rebalance.adoc#prerequisites[Prerequisites] in xref:manage:manage-nodes/modify-services-and-rebalance.adoc[Modify Services and Rebalance] before using the REST API.
+
+* For information about using the `POST /controller/rebalance` REST API to redistribute services data and processing after failovers or node additions and removal, see xref:rest-api:rest-cluster-rebalance.adoc[Rebalancing the Cluster].
```

Comment:
Realized that this sentence needed to be more clear.

* For information about using the `POST /controller/rebalance` REST API to rebalance after node additions and removals, after graceful failover & recovery, and after a hard failover, see xref:rest-api:rest-cluster-rebalance.adoc[Rebalancing the Cluster].